posthog/session_recordings/session_recording_api.py

```diff
@@ -845,6 +827,32 @@ def list_recordings(
     )
 
 
+def safely_read_modifiers_overrides(distinct_id: str, team: Team) -> HogQLQueryModifiers:
+    modifiers = HogQLQueryModifiers()
+
+    try:
+        groups = {"organization": str(team.organization.id)}
+        flag_key = "HOG_QL_ORG_QUERY_OVERRIDES"
+        flags_n_bags = posthoganalytics.get_all_flags_and_payloads(
+            distinct_id,
+            groups=groups,
+        )
+        # this loads nothing whereas the payload is available
+        # modifier_overrides = posthoganalytics.get_feature_flag_payload(
+        #     flag_key,
+        #     distinct_id,
+        #     groups=groups,
+        # )
+        modifier_overrides = (flags_n_bags or {}).get("featureFlagPayloads", {}).get(flag_key, None)
+        if modifier_overrides:
+            modifiers.optimizeJoinedFilters = json.loads(modifier_overrides).get("optimizeJoinedFilters", None)
+    except:
+        # be extra safe
```

Comment:
you're maybe overcorrecting 🙈 - but this is nice, defends against us putting in invalid payloads, and if I try uncommenting and testing the payload fn I wouldn't accidentally break replay